docs/products/clickhouse/howto/data-service-integration.md

```diff
@@ -1,134 +1,240 @@
-        service**.
-    1.  [Set up the new service](/docs/platform/howto/create_new_service).
-    1.  Come back to your primary service and create an integration to
-        the newly created service. For that purpose, skip the steps that
-        follow and start over with building your integration using this
-        instruction but now follow the steps below about **integrating
-        with an existing service**.
-
-    **Option 2: Use an existing service and integrate it**
-
-    To create an integration with an **existing service**:
-
-    1.  Select a service of the chosen type from the list of services
-        available for integration.
-    1.  Select **Continue** and proceed to the next step to integrate
-        the database.
-
-1.  In the **Integration databases** view, select either **Enable
-    without databases** or **Add databases** depending on whether you
-    want to enable your integration with databases:
-
-    -   To enable your integration **with** databases:
-        1.  In the **Integration databases** view, select **Add
-            databases**.
-
-        1.  In the **Add integration databases** section, enter database
-            names and schema names and select **Enable** when ready.
-
-            You can preview the created databases by selecting
-            **Databases tables** from the sidebar.
-    -   To enable your integration **without** databases
-        1.  In the **Integration databases** view, select **Enable
-            without databases**.
-
-            You can preview the created integration by selecting
-            **Overview** from the sidebar.
+   As a result, the **Data service integrations** wizard opens, showing
+   a list of all data sources available for integration. If there are no data sources to
+   integrate with, create an Aiven-managed service or an integration endpoint by
+   clicking **Create service** or **Add external endpoint** respectively.
+
+1. Depending on your use case, continue as instructed in either
+  [Integrate with Aiven-managed services](#integrate-aiven-services) or
+  [Integrate with external data sources](#integrate-external-services).
+
+### Integrate with Aiven-managed services {#integrate-aiven-services}
+
+You can integrate either with a new Aiven-managed service or with an existing Aiven-managed
+services:
+
+<Tabs groupId="group1">
+<TabItem value="1" label="Integrate with a new service" default>
+To create an integration with a new service in the **Data service integrations** wizard:
+
+1. Unselect all available data **Service types**.
+1. Click **Create service**.
+1. [Set up the new service](/docs/platform/howto/create_new_service).
+1. Come back to your Aiven for ClickHouse service (<ConsoleLabel name="overview"/> >
+   **Data pipeline** > **Add data source**), and create an integration to the newly created
+   service following up the instructions in next tab
+   **Integrate with an existing service**.
```

Comment:
Let's not describe the position of tabs, we might change their order one day, we shouldn't have to review the content doing so 👍 
```suggestion
   service following up the instructions in the
   **Integrate with an existing service** tab.
```